{{cookiecutter.project_shortname}}/scripts/bootstrap

```diff
@@ -3,13 +3,14 @@
 set -e
 
 pip install -r requirements.txt
-pip install -e .
+pip install -e .[all]
 
 npm update && npm install --silent -g node-sass@3.8.0 clean-css@3.4.19 uglify-js@2.7.3 requirejs@2.2.0
 
 CWD=`pwd`
 invenio npm
-cd ${VIRTUAL_ENV}/var/instance/static
+python_prefix=$(python -c "import sys; print(sys.prefix)")
```

Comment:
Actually, I added this because of Docker (and having `VIRTUAL_ENV` set, basically means that you have a different python prefix already). On OpenShift it would actually fail, since the directory that holds the static files is defined by `INVENIO_INSTANCE_PATH`.

The real problem is that `npm install` should be running as part of the application with something like `invenio npm install` and [`pynpm`](https://github.com/inveniosoftware/pynpm), so that `current_app.static_folder` can be used.